package.json

```diff
@@ -489,12 +489,35 @@
       {
         "type": "cmake",
         "required": [
-          "command"
+          "command",
+          "label"
```

Comment:
>  "label" [](http://example.com/codeflow?start=8&length=9)

Since we are modifying the existing task provider, we can't make this required without breaking people.  We should have the label be optional (it is optional for tasks.json in general) and create our own label for it when it is not set (e.g. "cmake: build")